fr/docs/sso/index.md

```
@@ -1,12 +1,12 @@
-# SSO Integration Guide for Mammouth AI Business customers
+# SSO Integration Guide for Mammouth AI Business customers (per-IdP setup instructions)
 
 This document is for IT administrators of organizations that want to enable Single Sign-On (SSO) for their users on [mammouth.ai](https://mammouth.ai).
 
-You can find the Mammouth AI logo to add in your integrations here https://public.mammouth.ai/
+You can find the Mammouth AI logo to add in your integrations [here](../../docs/branding-kit/index.md)
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)